tools/dtrace/upstairs_count.d

```diff
@@ -0,0 +1,58 @@
+/*
+ * Display internal Upstairs live repair status.
+ */
+#pragma D option quiet
+#pragma D option strsize=1k
```

Comment:
brilliant!

Reply:
It should be noted that I called the DTrace hotline and a friendly operator was standing by and helped me solve my DTrace issues.